sdcm/utils/sstable/sstable_utils.py

```diff
@@ -64,6 +65,7 @@ def is_sstable_encrypted(self, sstables=None) -> list:
 
                 sstables_res = self.db_node.remoter.sudo(
                     f"{self.db_node.add_install_prefix('/usr/bin/scylla')} sstable dump-scylla-metadata"
+                    f" --scylla-yaml-file {self.db_node.add_install_prefix(SCYLLA_YAML_PATH)}"
```

Comment:
Add comment referring to the existing bug: https://github.com/scylladb/scylla-enterprise/issues/3461 ?

Reply:
The issue is not aligned for 100%, ok.

Is there another issue which describes the error we get not using the `--scylla-yaml-file`?
Why do we need to add the `--scylla-yaml-file` flag?

Reply:
looks at the scylla code there is automatic detection of the schema, the scylla yaml is the last option, not the first.
so it might find the schema, but then wouldn't have loaded the configuration for the encryption to work.
@denesb, we might want to consider scylla.yml to be the default ?

so for encryption to work, we have to specific the configuration. as we now do in dtest as well.

Reply:
Yes, I can make `scylla.yaml` the default. Let me cook a patch.

Reply:
Opened an issue to track this effort: https://github.com/scylladb/scylladb/issues/16132

The change is not trivial, but it is not difficult either.

Reply:
thanks @denesb